README.md

```diff
@@ -70,41 +69,33 @@ Want to play around? [![Try gurmukhi-utils on RunKit](https://badge.runkitcdn.co
 
 ## API
 
-### firstLetters(line, [stripNukta], [withVishraams]) ⇒ <code>String</code>
-Generates the first letters for a given ASCII or unicode Gurmukhi string.
By default, the function will transform letters with bindi to their simple equivalent,
for example, zaza to jaja (ਜ਼ => ਜ).
+### firstLetters(line) ⇒ <code>String</code>
+Generates the first letters for a unicode Gurmukhi,
+Hindi transliteration, Shahmukhi transliteration, or English transliteration string.
+Includes any end-word vishraams, and line-end characters.
```

Comment:
If including line end characters, won't it require the user to type them in for an exact match? In relevance to search result ordering prioritization

Reply:
No, because we’ll use `stripEndings()` with this function in database.